docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -76,7 +76,14 @@ Property Name                                Description
                                              the object has been created.
 
 ``hive.s3.upload-acl-type``                  Canned ACL to use while uploading files to S3, defaults
-                                             to ``Private``.
+                                             to ``PRIVATE``. If the files are to be uploaded to a
+                                             S3 bucket owned by a different AWS user, the canned ACL should be
+                                             set to one of the following: ``AUTHENTICATED_READ``,
+                                             ``AWS_EXEC_READ``, ``BUCKET_OWNER_FULL_CONTROL``, ``BUCKET_OWNER_READ``,
+                                             ``LOG_DELIVERY_WRITE``, ``PUBLIC_READ`` , ``PUBLIC_READ_WRITE``.
+                                             Please refer to the `AWS canned ACL <https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-s3-acls.html>`_
+                                             guide to understand each option's definition. Additionally, the property 
+                                             ``hive.s3-file-system-type`` needs to be set to ``TRINO``.
```

Comment:
Should this property also be documented separately?

Reply:
Thanks for the feedback @mosabua, for some context about that particular sentence: I am currently using Trino version 338 and the default value was set to `hive.s3-file-system-type = EMRFS`. If I remember correctly, the `EMRFS` is now deprecated and `TRINO` is the default in the latest release so the last sentence might not be needed. Do you think it would be better if I left this out altogether?

Reply:
I dont know the technical details enough .. @findepi ?